src/Controls/src/Core/Platform/Android/AutomationPropertiesProvider.cs

```diff
@@ -143,6 +143,12 @@ internal static void SetAutomationId(AView control, string automationId, string
 			SetBasicContentDescription(control, element, defaultContentDescription);
 		}
 
+		internal static void SetImportantForAccessibility(AView control, Element element)
+		{
+			ImportantForAccessibility? _defaultImportantForAccessibility = null;
```

Comment:
Should we maybe set this to ImportantForAccessibility.Auto instead of null?

Reply:
This only exists to satisfy calling `SetImportantForAccessibility` in a scenario where we don't want to effect a default. We only want to change ImportantForAccessibility to whatever the user has indicated. If we want to try and set an accurate default then we'll need to copy this super fun method from XF


In XF the way this worked was to grab the current setting from the control and then use that from the Default but that's not very accurate. From what I remember even just setting it to Auto can be a bit weird.

That's why this monstrosity exists.... Once you've set important for accessibility it's your responsibility to always set it to what you want. It's the same as the `Color.Default` conversations

https://github.com/xamarin/Xamarin.Forms/blob/42f059586b6a4f2d79c102c8fd22cd6a5bf0f7e3/Xamarin.Forms.Platform.Android/FastRenderers/AutomationPropertiesProvider.cs#L35